ingestion_program/ingestion_parallel.py

```diff
@@ -383,7 +384,7 @@ def fit_submission(self):
         print("[*] Calling fit method of submitted model")
         self.model.fit()
 
-    def predict_submission(self, test_settings):
+    def predict_submission(self, test_settings, initial_seed=31415):
```

Comment:
We should probably pull 31415 out as a constant as its used in two places.

Reply:
So we use a different number ?

Reply:
Please check now